server/src/db/migrations/20210802155325_addSetsTable.cjs

```diff
@@ -0,0 +1,34 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("sets", (table) => {
+        table.bigIncrements("id")
+
+        table.string("name").notNullable()
+        table.string("profile").notNullable()
+        table.string("imageURLpath").notNullable()
+
+        table.string("link")
+        table.string("designer")
+        table.string("vendor")
+        table.date("releaseDate")
+        table.string("salesFormat")
+        table.string("round")
+        table.string("status")
```

Comment:
love the amount of columns. The more of these we have, the more we can do with this interesting data

Do you think you will need to make a vendor and designer table as the app grows?